dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -180,7 +179,7 @@ GetFileTypeIconsByKey(HKEY hKey, PFILE_TYPE_ENTRY Entry)
 static BOOL
 QueryFileDescription(LPCWSTR ProgramPath, LPWSTR pszName, INT cchName)
 {
-    SHFILEINFOW FileInfo = { 0 };
+    SHFILEINFOW FileInfo;
```

Comment:
Why did you remove the initialization?

Reply:
It doesn't use the icon handle in here. And the return value of SHGetFileInfo will be checked. So SHFILEINFO doesn't require initialization.

Reply:
Revert or not [R/n]?

Reply:
OK.